app/src/main/java/com/dsvag/currencyexchanger/data/database/CoinDAO.kt

```diff
@@ -0,0 +1,19 @@
+package com.dsvag.currencyexchanger.data.database
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.Query
+import com.dsvag.currencyexchanger.data.models.latest.Coin
+
+@Dao
+interface CoinDAO {
+    @Insert
+    fun insertAll(vararg coin: Coin)
+
+    @Delete
+    fun deleteAll(vararg coin: Coin)
+
+    @Query("SELECT * FROM Coin")
+    fun getCoins(): List<Coin>
```

Comment:
Когда будешь использовать возвращай не просто список, а `Single` или `Flowable`
```suggestion
    fun getCoins(): Single<List<Coin>>
```